### ISSUE TITLE
fsouza/go-dockerclient using `pull` command doesn't work on latest version of docker

### ISSUE THREAD

**Issue by KolbyML** [ethereum/hive#1006]
https://github.com/fsouza/go-dockerclient/issues/1030

^ this post goes over it
```
API error (400): client version 1.16 is too old. Minimum supported API version is 1.24, please upgrade your client to a newer version
```
^ here is an error from my system when I use `./hive --docker.pull xyc...`

The reason I found this was because portal-hive.ethdevops.io the clients haven't updated since the 16th so I guess I will need to ask paul to downgrade the version of docker we use

*TL:DR* (written later)
We had to downgrade our version of docker because of the go-docker library hive uses only supports docker pull version 1.16, but the latest release of docker requires 1.24 minimum. So I asked Paul to downgrade our docker version and it worked

**Comment by fjl** [comment]
I have upgraded the go-dockerclient dependency, so maybe this is resolved now?

**Comment by KolbyML** [comment]
Updating go-dockerclient won't fix it https://github.com/fsouza/go-dockerclient/blob/594f32e0658177fe731a06931affceabf3594f2b/image.go#L516 as seen here they only support 1.16 even on the latest version of the library

**Comment by fjl** [comment]
It should be fixed by #1125

**Comment by KolbyML** [comment]
Ok I will test the PR 👍
